core/payment/src/cli.rs

```diff
@@ -18,12 +18,17 @@ pub enum PaymentCli {
         #[structopt(long, default_value = DEFAULT_PAYMENT_DRIVER)]
         driver: String,
         #[structopt(long)]
-        network: Option<String>,
+        network: Option<String>, // TODO: network enum
```

Comment:
Payment driver implementors will have absolute freedom defining networks and tokens. There's **not** going to be a network enum nor token enum.

Reply:
I'd argue but not at this stage. :)